### DISCUSSION TITLE
Help:A question about connecting to Milo server and reporting errors

### DISCUSSION THREAD

**Discussion by sunzbin** [eclipse-milo/milo#1301] [General]
I built an opcua service using milo, but when using client loop connection, if there is no sleep between client threads, only four connections can be successfully connected, and the rest will fail. What is the reason for this?
![daim](https://github.com/user-attachments/assets/15b4ae92-a91c-419b-adc3-5fe9b66c6ea0)
![log](https://github.com/user-attachments/assets/44293de8-5ff2-4987-aec0-48e47dce8cb2)

**Comment by kevinherron** [comment]
What server are you connecting to?

**Reply by sunzbin** [reply]
OPC server  I ran using the example code from Eclipse Milo

**Reply by kevinherron** [reply]
The server has rate limiting enabled by default, it may be the issue.
